pkg/apis/kops/componentconfig.go

```diff
@@ -292,6 +292,29 @@ type KubeControllerManagerConfig struct {
 	UseServiceAccountCredentials *bool `json:"useServiceAccountCredentials,omitempty" flag:"use-service-account-credentials"`
 }
 
+// CloudControllerManagerConfig is the configuration for the controller
+type CloudControllerManagerConfig struct {
+	//Master is the url for the kube api master
+	Master string `json:"master,omitempty" flag:"master"`
```

Comment:
Where are these flags going to?  k-c-m, or is this a new pod?

Reply:
We will need validation for these new API elements.  See https://github.com/kubernetes/kops/blob/master/pkg/apis/kops/validation/validation.go

Reply:
What do you think about dropping the `Config` from `CloudControllerManagerConfig`?  Before you make changes let's get a couple more contributors looking at it.

Reply:
If we are using an external provider for say AWS, what happens to the config manager stuff?  For example, we are able to set which security group the AWS cloud provider uses.  We have this functionality in kops already.  Do these changes impact that?

Reply:
Isn't the config centrally stored in S3? If KCM can pull it, then CCM can also pull it.